Tasks/XamariniOS/xamarinios.ts

```diff
@@ -152,7 +101,7 @@ async function run() {
             buildRunner.line(args);
         }
         buildRunner.argIf(codesignKeychain, '/p:CodesignKeychain=' + codesignKeychain);
```

Comment:
I think this line isn't used anymore.

Reply:
Good point, we decided not to expose this in the task inputs for the Xcode task as well. I will remove it.